util/configure

```diff
@@ -359,6 +369,12 @@ for my $opt (@ARGV) {
     }
 }
 
+for my $lib (qw(openssl pcre zlib)) {
+    if (exists($ngx_libs{$lib}) && not exists($ngx_libs_opt{$lib."-opt"})) {
+        push @ngx_opts, "--with-".$lib."-opt=-g";
```

Comment:
Please always use spaces around binary operators like `.`.

Reply:
The logic here does not look quite right. Even if the user specifies her own `--with-xxx-opt`, we should still prepend `-g` to the option string.